perfTest/recreateDb.js

```diff
@@ -0,0 +1,28 @@
+const { Pool } = require("pg");
+const { parse } = require("pg-connection-string");
+
+if (!process.env.PERF_DATABASE_URL) {
+  throw new Error(
+    "No PERF_DATABASE_URL setting detected; please don't call this script directly!"
+  );
+}
+const config = parse(process.env.PERF_DATABASE_URL);
+// don't connect to the provided db, or we can't drop it
+const pgPool = new Pool({ ...config, database: "postgres" });
```

Comment:
Not everyone has a `postgres` database (I learned this the hard way), so I think using `template1` is more compatible:

```suggestion
const pgPool = new Pool({ ...config, database: "template1" });
```

Reply:
Yeah, but when I tried this it doesn’t work in docker - I’m not sure why. See https://github.com/graphile/worker/pull/64#discussion_r367818939

I guess we might need to use template1 only if not running in docker? Feels a bit messy but can’t think of another solution right now. Let me know if you have any ideas

Reply:
Actually, the error I was seeing has gone away now that I'm using a single client as suggested. So all good

Reply:
Ah; ignore my reply above - this is exactly what I figured was going on :+1: